app/src/main/java/de/westnordost/streetcomplete/quests/drinking_water_type/DrinkingWaterType.kt

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.drinking_water_type
+
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.hasCheckDate
+import de.westnordost.streetcomplete.osm.updateCheckDate
+import de.westnordost.streetcomplete.quests.drinking_water_type.DrinkingWaterType.*
+
+enum class DrinkingWaterType {
+    WATER_TAP,
+    WATER_FOUNTAIN_JET,
+    WATER_FOUNTAIN_BOTTLE_REFILL_ONLY,
+    WATER_FOUNTAIN_GENERIC,
+    HAND_PUMP,
+    WATER_WELL_WITHOUT_PUMP,
+    SPRING,
+    DISUSED_DRINKING_WATER,
+}
+
+fun DrinkingWaterType.applyTo(tags: Tags) {
+    when (this) {
+        WATER_FOUNTAIN_GENERIC -> tags["fountain"] = "drinking"
+        WATER_FOUNTAIN_JET -> tags["fountain"] = "bubbler"
+        WATER_FOUNTAIN_BOTTLE_REFILL_ONLY -> tags["fountain"] = "bottle_refill"
+        WATER_TAP -> tags["man_made"] = "water_tap"
```

Comment:
Please reorder to match order above (this got lost in some earlier comment apparently):

```suggestion
        WATER_TAP -> tags["man_made"] = "water_tap"
        WATER_FOUNTAIN_JET -> tags["fountain"] = "bubbler"
        WATER_FOUNTAIN_BOTTLE_REFILL_ONLY -> tags["fountain"] = "bottle_refill"
        WATER_FOUNTAIN_GENERIC -> tags["fountain"] = "drinking"
```